src/ports/postgres/modules/deep_learning/test/unit_tests/test_madlib_keras.py_in

```diff
@@ -1259,6 +1259,26 @@ class MadlibKerasWrapperTestCase(unittest.TestCase):
             self.subject.parse_and_validate_compile_params(test_str)
         self.assertIn('invalid optimizer', str(error.exception))
 
+    def test_parse_callbacks_pass(self):
+        test_str = """'[TensorBoard(log_dir="/tmp/logs/fit")]'"""
```

Comment:
maybe also add a couple of other tensorboard params here other than log_dir. If it's not straightforward, then feel free to skip it

Reply:
Looks like you asked this in another comment already.  I agree with @orhankislal 's response:

"I don't think we have to test for every keras parameter combination. The log_dir is tested because it is error-prone with arbitrary paths but a numeric value like batch_size either works or keras throws an appropriate error."